### ISSUE TITLE
Question: ...Can't change webUI port

### ISSUE THREAD

**Issue by MrSoupman** [ArchiveBox/ArchiveBox#1571]
Hello,

I set up ArchiveBox as outlined in the quickstart guide and confirmed I was able to connect to the webUI on port 8000. However, I have another docker container that's listening on port 8000, so I wanted to change the port the ArchiveBox's webUI listens on. At first, I just modified the ports field in the docker-compose.yml from `8000:8000` to `3000:3000`. Attempting to connect to the webUI gives me an unable to connect error.
I checked `docker ps` and that showed Archivebox is running on `0.0.0.0:3000->3000/tcp, :::3000->3000/tcp, 8000/tcp`.
I brought the container down and edited ArchiveBox.conf to include `BIND_ADDR = "0.0.0.0:3000"` but still nothing. 
Then I ran `docker compose run archivebox config --set BIND_ADDR="['127.0.0.1:3000', '0.0.0.0:3000'][SHELL_CONFIG.IN_DOCKER]"` in an attempt to force the binding address to set to the correct port. Still no luck.

I tried switching from latest to dev but no luck. Switching the ports back to 8000 on the compose file allowed me to connect to the webUI, and the configurations page showed that BIND_ADDR was changed.
Am I missing a configuration item or did I miss something in the documentation? I didn't find anything really related to changing the port of the webUI.

**Comment by MrSoupman** [comment]
Okay, I did a bit more reading on docker containers and figured I set the ports incorrectly. I had to modify the compose file with respect to Docker's HOST_PORT and CONTAINER_PORT distinction.

TL;DR: If I want to change the webUI's port to 3000, I should have modified the ports field to be `3000:8000` instead.
